Packages/io.chainsafe.web3-unity/Editor/ServerSettings.cs

```diff
@@ -196,6 +199,18 @@ private void OnGUI()
         chainID = EditorGUILayout.TextField("Chain ID: ", chainID);
         symbol = EditorGUILayout.TextField("Symbol: ", symbol);
         blockExplorerUrl = EditorGUILayout.TextField("Block Explorer: ", blockExplorerUrl);
+        analyticsOptOut = EditorGUILayout.Toggle("Collect Data for Analytics: ", analyticsOptOut);
```

Comment:
please change the analyticsOptOut to enableAnalytics across the files since this name is not in line with the logic, but it's quite the opposite.

Reply:
and set the default value to true ofc.

Reply:
moment

Reply:
done